content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -3,399 +3,1025 @@ title: Using Windows Server Containers in Kubernetes
 toc_hide: true
 ---
 {{< note >}}
-These instructions were recently updated based on Windows Server platform enhancements and the Kubernetes v1.9 release
+These instructions are under review for the v1.14 release with a [tracking issue](https://github.com/kubernetes/website/issues/12426).
```

Comment:
LGTM in general. 

One nit: the content of this file seems too long. We may split the contents into different files, e.g. index for general introdution and setup/usage/features in different files.

Reply:
This has been the plan with these docs in agreement with sig-docs to submit as one large document, and then figure out if it should be divided and where the parts should land